error: [Prusti: verification error] postcondition might not hold.
  --> $DIR/enums-1.rs:18:11
   |
18 | #[ensures(result)]
   |           ^^^^^^
   |
note: the error originates here
  --> $DIR/enums-1.rs:19:1
   |
19 | / fn test1(x: OneOfFive, y: SecondEnum) -> bool {
20 | |     match x {
21 | |         OneOfFive::One(_) => true,
22 | |         OneOfFive::Five { a, b, c, d, e } => match y {
...  |
27 | |     }
28 | | }
   | |_^
note: counterexample for "x"
  initial value: OneOfFive::Five {
      a: ?,
      b: ?,
      c: ?,
      d: ?,
      e: ?,
  }
  final value:   OneOfFive::Five {
      a: ?,
      b: ?,
      c: ?,
      d: ?,
      e: ?,
  }
  --> $DIR/enums-1.rs:19:10
   |
19 | fn test1(x: OneOfFive, y: SecondEnum) -> bool {
   |          ^
note: counterexample for "y"
  initial value: SecondEnum::One
  final value:   SecondEnum::One
  --> $DIR/enums-1.rs:19:24
   |
19 | fn test1(x: OneOfFive, y: SecondEnum) -> bool {
   |                        ^
note: counterexample for "a"
  final value:   4
  --> $DIR/enums-1.rs:22:27
   |
22 |         OneOfFive::Five { a, b, c, d, e } => match y {
   |                           ^
note: counterexample for "b"
  final value:   true
  --> $DIR/enums-1.rs:22:30
   |
22 |         OneOfFive::Five { a, b, c, d, e } => match y {
   |                              ^
note: counterexample for "c"
  final value:   4
  --> $DIR/enums-1.rs:22:33
   |
22 |         OneOfFive::Five { a, b, c, d, e } => match y {
   |                                 ^
note: counterexample for "d"
  final value:   0x4
  --> $DIR/enums-1.rs:22:36
   |
22 |         OneOfFive::Five { a, b, c, d, e } => match y {
   |                                    ^
note: counterexample for "e"
  final value:   false
  --> $DIR/enums-1.rs:22:39
   |
22 |         OneOfFive::Five { a, b, c, d, e } => match y {
   |                                       ^
note: counterexample for result
  final value:   false
  --> $DIR/enums-1.rs:19:42
   |
19 | fn test1(x: OneOfFive, y: SecondEnum) -> bool {
   |                                          ^^^^

error: aborting due to previous error

